tests/integration/helpers.py

```diff
@@ -408,7 +408,7 @@ def get_workflow_runs(
         latest_job: WorkflowJob = run.jobs()[0]
         logs = get_job_logs(job=latest_job)
 
-        if JOB_LOG_START_MSG_TEMPLATE.format(runner_name=runner_name) in logs:
+        if runner_name in logs:
```

Comment:
I believe that `JOB_LOG_START_MSG_TEMPLATE` is now unused? We really need to have a look at the linting, it seems to be either not triggered or misconfigured.

I assume that there cannot be any false positives with this method (so that a job has not yet started although the runner name is in the logs), as the string previously explicitly contained `Job is about to start on the runner:`

Reply:
It seems the `JOB_LOG_START_MSG_TEMPLATE` is used in one more place, which is why the linting is not triggered.

Yanks has mention he is looking into why some test are passing when `Job is about to start on the runner:` is not in the message anymore.

I do think fixing those tests are outside the scope of this PR.